### DISCUSSION TITLE
i want to make background video and have problem!

### DISCUSSION THREAD

**Discussion by edanTavory** [vercel/next.js#37666] [App Router]
Error: Hydration failed because the initial UI does not match what was rendered on the server.

import Head from 'next/head'
import styles from '../styles/Home.module.scss'
import ReactPlayer from 'react-player'
import { useEffect, useState } from 'react'

export default function Home() {
  const [video,setVideo]=useState("")
  useEffect(()=>setVideo("public\House1.mp4"),[])
  return (
    <div className={styles.container}>
      <Head>
        <title>Create Next App</title>
        <meta name="description" content="Generated by create next app" />
        <link rel="icon" href="/favicon.ico" />
      </Head>

      <main className={styles.main}>
   <div>
<ReactPlayer src={video} type='video/mp4' />
       

   </div>
      </main>

    </div>
  )
}

**Comment by wiggledom** [comment]
You can try with the following codes

import React, { useEffect } from "react";
import VideoPlayer from "react-background-video-player";
import vid0 from "../video/static.mp4";

....

<VideoPlayer
            className="video"
            src={vid0}
            autoPlay={true}
            volume={1}
            muted={false}
/>


I 'd appreciate it if this code can help to solve your problem

**Reply by wiggledom** [reply]
< VideoPlayer
     className="video"
     src={vid0}
     autoPlay={true}
     volume={1}
     muted={true}
 />

This is source code

**Comment by OtabekVaxobov** [comment]
`
<video autoPlay muted loop className="video">
          <source src="/video.mp4" type="video/mp4" />
        </video>`

**Reply by OtabekVaxobov** [reply]
`.video {
  object-fit: cover;
  width: 100%;
  height: 100%;
  position: relative;
  z-index: -1;
}`
